src/user/user.exceptions.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+
+export class UserAlreadyExists extends HttpException {
```

Comment:
With this naming it is not clear that UserAlreadyExists is an Exception. Same for UserNotActivated. I would add the Exception "keyword" to it to clearly indicate what kind of class this is.

Reply:
Fixed in #49